src/Test/Utilities/Portable/Compilation/OperationTreeVerifier.cs

```diff
@@ -65,6 +65,9 @@ private void LogCommonPropertiesAndNewLine(IOperation operation)
         {
             LogString(" (");
 
+            // Child Ordial
+            LogString($"[{GetChildOrdinal(operation)}] ");
```

Comment:
:bulb: This would be better implemented as a change on its own. Also, since this change is intended to highlight the behavior change in #22726, initially implement the change against the master branch *before* #22726 is merged, showing the original order. Then merge the current master branch into the PR branch, and update the tests to show the new order.

Reply:
oh.. well.. sure..